Update assumptions scoring in the main readme

lizgzil opened the pull request:
---

# Description

Please include a summary of the changes.

Fixes # (issue)

# Instructions for Reviewer

In order to test the code in this PR you need to ...

Please pay special attention to ...

# Checklist:

- [ ] I have refactored my code out from `notebooks/`
- [ ] I have checked the code runs
- [ ] I have tested the code
- [ ] I have run `pre-commit` and addressed any issues not automatically fixed
- [ ] I have merged any new changes from `dev`
- [ ] I have documented the code
  - [ ] Major functions have docstrings
  - [ ] Appropriate information has been added to `README`s
- [ ] I have explained this PR above
- [ ] I have requested a code review

lizgzil committed:
Update assumptions scoring in the main readme

helloaidank commented:
> # Description
> Please include a summary of the changes.
> 
> Fixes # (issue)
> 
> # Instructions for Reviewer
> In order to test the code in this PR you need to ...
> 
> Please pay special attention to ...
> 
> # Checklist:
> * [ ]  I have refactored my code out from `notebooks/`
> * [ ]  I have checked the code runs
> * [ ]  I have tested the code
> * [ ]  I have run `pre-commit` and addressed any issues not automatically fixed
> * [ ]  I have merged any new changes from `dev`
> * [ ]  I have documented the code
>   
>   * [ ]  Major functions have docstrings
>   * [ ]  Appropriate information has been added to `README`s
> * [ ]  I have explained this PR above
> * [ ]  I have requested a code review

Hey Thanks for this! I just have a couple of questions, more with regard to the assumptions!

- We are making the assumption that an area has to be urban to have a heat network, just wondering if there are any edge cases where we hit a certain property density within an area that it makes sense to have a heat network there provided there is an anchor property of some sort? 
- Within that, if the heat network is powered by a large heat pump, in the standard view would the grid still need the capacity to support this? 
- When referring to GSHP here, we are then only referring to borehole GSHP as those with ground loops require digging a big old trench that is 100-200 m long.
- I am guessing we should ignore the maximum points scores here for those with x! 

Looks all good otherwise!

crispy-wonton commented on README.md:
```diff
@@ -63,19 +63,21 @@ heat pump suitability scores are calculated in total per LSOA. Scores are first
+|                                                                                                  | ASHP (S) | ASHP (N) | GSHP (S) | GSHP (N) | SGL (S) | SGL (N) | HN (S) | HN (N) |
+| ------------------------------------------------------------------------------------------------ | -------- | -------- | -------- | -------- | ------- | ------- | ------ | ------ |
+| Is the property NOT listed?                                                                      | 0.25     | 0.25     | 0.25     | 0.25     | 0.25    | 0.25    | 0.25   | 0.25   |
+| Is the property NOT in a protected area?                                                         | 0.25     | 0.25     | 0.25     | 0.25     | 0.25    | 0.25    | 0.25   | 0.25   |
```
We could put an asterisk here and specify below that "protected area" refers to building conservation zones in England and Wales and World Heritage Sites in Scotland.

crispy-wonton commented on README.md:
```diff
@@ -63,19 +63,21 @@ heat pump suitability scores are calculated in total per LSOA. Scores are first
+| Is this property in a LSOA with a high property density? (> `60` households per km2)             | 0        | 0        | 0        | 0        | 2       | 2       | 0      | 0      |
+| Is this property in an urban LSOA?                                                               | 0        | 0        | 0        | 0        | 0       | 0       | 2      | 2      |
+| Is this property in a LSOA with an anchor property?                                              | 0        | 0        | 0        | 0        | 0       | 0       | 1      | 1      |
+| What proportion of properties in this LSOA could the electricity grid support to have HPs? (`x`) | `x`      | 0        | `x`      | 0        | `x`     | 0       | 0      | 0      |
```
We could specify here that x ranges 0-1

crispy-wonton reviewed (commented):
Left a couple of minor suggestions but it all looks correct. I have double checked the scores against the `pipeline/suitability/scoring.py` file and have checked the totals. All looks good.

lizgzil committed:
Amendments to readme scoring assumptions after Aidan and Roisin's review

lizgzil commented:
Thanks @helloaidank and @crispy-wonton (and sorry Aidan - I have only just seen your review!). I made some changes based off your comments.

@helloaidank 

> - We are making the assumption that an area has to be urban to have a heat network, just wondering if there are any edge cases where we hit a certain property density within an area that it makes sense to have a heat network there provided there is an anchor property of some sort?

Really good point, I think it might be worth understanding the correlation between urban/rural and property density to explore these edge cases.

> - Within that, if the heat network is powered by a large heat pump, in the standard view would the grid still need the capacity to support this?

Another good point and doesnt feel like this would be too hard to add. I've added [this issue](https://github.com/nestauk/asf_heat_pump_suitability/issues/119).

> - When referring to GSHP here, we are then only referring to borehole GSHP as those with ground loops require digging a big old trench that is 100-200 m long.

Yes I think so.

> - I am guessing we should ignore the maximum points scores here for those with x!

The max points are where x=1. I've added some clarification about this to make it clearer.
